app/src/main/java/com/example/parkingsystemjava/fragments/SpacesSettingDialogFragment.java

```diff
@@ -0,0 +1,56 @@
+package com.example.parkingsystemjava.fragments;
+
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.example.parkingsystemjava.databinding.DialogFragmentParkingSpacesSettingBinding;
+import com.example.parkingsystemjava.listener.ListenerDialogFragment;
+import com.example.parkingsystemjava.mvp.contract.ParkingSpacesContract;
+import com.example.parkingsystemjava.mvp.presenter.ParkingSpacesPresenter;
+import com.example.parkingsystemjava.mvp.view.ParkingSpacesView;
+
+public class SpacesSettingDialogFragment extends DialogFragment {
+
+    private static final String LISTENER_KEY = "LISTENER_KEY";
+
+    private DialogFragmentParkingSpacesSettingBinding binding;
+    private ParkingSpacesContract.ParkingSpacesDialogFragmentPresenter presenter;
+    private ParkingSpacesContract.ParkingSpacesDialogFragmentView view;
```

Comment:
this value is not necessary, remove it please

Reply:
done